torch/_subclasses/fake_tensor.py

```diff
@@ -635,6 +635,9 @@ def __torch_dispatch__(cls, func, types, args=(), kwargs=None):
                     assert fake_mode is arg.fake_mode, "Mixing modes NYI"
 
         assert fake_mode is not None
+        if hasattr(fake_mode, "tracking") and fake_mode.tracking.on_stack:
```

Comment:
could you add a comment explaining this ?

Reply:
Added